python/hsfs/feature_view.py

```diff
@@ -76,6 +77,15 @@ def __init__(
         self._single_vector_server = None
         self._batch_vectors_server = None
         self._batch_scoring_server = None
+        if id:
```

Comment:
will there be a `enable_feature_monitoring` method in FV and FG?

Reply:
Ja, something of the sort, but we still don't have a good UX design for it